Library/Formula/xhyve.rb

```diff
@@ -0,0 +1,18 @@
+class Xhyve < Formula
+  desc "xhyve, a lightweight OS X virtualization solution"
+  homepage "http://www.pagetable.com/?p=831"
+  url "https://github.com/mist64/xhyve/archive/84868be7a65305a3a7ab8f3c197d6df93ebbe5dc.tar.gz"
+  version "0.0.0"
```

Comment:
@dguerri If upstream releases a stable one it’ll fit in the core, but if it’s an alpha version it’ll go in [Homebrew/devel-only](https://github.com/Homebrew/homebrew-devel-only).